[2025-06-19T01:17:19.331+0000] {processor.py:161} INFO - Started process (PID=51397) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:17:19.332+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:17:19.333+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:17:19.333+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:17:19.336+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:17:19.335+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:17:19.337+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:17:19.352+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.029 seconds
[2025-06-19T01:17:49.840+0000] {processor.py:161} INFO - Started process (PID=51433) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:17:49.841+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:17:49.842+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:17:49.842+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:17:49.845+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:17:49.844+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:17:49.846+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:17:49.859+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.026 seconds
[2025-06-19T01:18:20.351+0000] {processor.py:161} INFO - Started process (PID=51469) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:18:20.352+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:18:20.353+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:18:20.352+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:18:20.355+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:18:20.354+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:18:20.356+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:18:20.370+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.027 seconds
[2025-06-19T01:18:51.146+0000] {processor.py:161} INFO - Started process (PID=51511) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:18:51.147+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:18:51.148+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:18:51.148+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:18:51.151+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:18:51.150+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:18:51.151+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:18:51.169+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.031 seconds
[2025-06-19T01:19:21.782+0000] {processor.py:161} INFO - Started process (PID=51547) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:19:21.784+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:19:21.785+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:19:21.784+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:19:21.788+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:19:21.786+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:19:21.788+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:19:21.805+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.031 seconds
[2025-06-19T01:19:52.304+0000] {processor.py:161} INFO - Started process (PID=51583) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:19:52.305+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:19:52.306+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:19:52.306+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:19:52.309+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:19:52.308+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:19:52.309+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:19:52.323+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.026 seconds
[2025-06-19T01:20:22.737+0000] {processor.py:161} INFO - Started process (PID=51619) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:20:22.738+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:20:22.739+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:20:22.739+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:20:22.742+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:20:22.741+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:20:22.742+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:20:22.757+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.027 seconds
[2025-06-19T01:20:53.156+0000] {processor.py:161} INFO - Started process (PID=51655) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:20:53.158+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:20:53.159+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:20:53.158+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:20:53.162+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:20:53.161+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:20:53.163+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:20:53.177+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.028 seconds
[2025-06-19T01:21:24.035+0000] {processor.py:161} INFO - Started process (PID=51691) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:21:24.036+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:21:24.037+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:21:24.037+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:21:24.039+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:21:24.039+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:21:24.040+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:21:24.057+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.030 seconds
[2025-06-19T01:21:54.637+0000] {processor.py:161} INFO - Started process (PID=51727) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:21:54.638+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:21:54.639+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:21:54.638+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:21:54.641+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:21:54.640+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:21:54.642+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:21:54.656+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.027 seconds
[2025-06-19T01:22:25.211+0000] {processor.py:161} INFO - Started process (PID=51763) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:22:25.213+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:22:25.214+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:22:25.214+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:22:25.216+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:22:25.216+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:22:25.217+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:22:25.231+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.027 seconds
[2025-06-19T01:22:56.071+0000] {processor.py:161} INFO - Started process (PID=51799) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:22:56.072+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:22:56.073+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:22:56.072+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:22:56.075+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:22:56.075+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:22:56.076+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:22:56.092+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.029 seconds
[2025-06-19T01:23:26.562+0000] {processor.py:161} INFO - Started process (PID=51835) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:23:26.562+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:23:26.564+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:23:26.564+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:23:26.567+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:23:26.566+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:23:26.569+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:23:26.585+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.031 seconds
[2025-06-19T01:23:57.288+0000] {processor.py:161} INFO - Started process (PID=51871) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:23:57.289+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:23:57.290+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:23:57.290+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:23:57.293+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:23:57.292+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:23:57.294+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:23:57.309+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.028 seconds
[2025-06-19T01:24:27.871+0000] {processor.py:161} INFO - Started process (PID=51907) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:24:27.872+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:24:27.874+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:24:27.873+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:24:27.876+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:24:27.876+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:24:27.877+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:24:27.891+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.027 seconds
[2025-06-19T01:24:58.382+0000] {processor.py:161} INFO - Started process (PID=51943) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:24:58.383+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:24:58.384+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:24:58.384+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:24:58.387+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:24:58.386+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:24:58.388+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:24:58.412+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.038 seconds
[2025-06-19T01:25:29.057+0000] {processor.py:161} INFO - Started process (PID=51979) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:25:29.058+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:25:29.059+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:25:29.058+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:25:29.061+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:25:29.060+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:25:29.062+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:25:29.077+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.028 seconds
[2025-06-19T01:25:59.621+0000] {processor.py:161} INFO - Started process (PID=52015) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:25:59.622+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:25:59.623+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:25:59.623+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:25:59.626+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:25:59.625+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:25:59.626+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:25:59.640+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.026 seconds
[2025-06-19T01:26:30.436+0000] {processor.py:161} INFO - Started process (PID=52051) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:26:30.438+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:26:30.439+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:26:30.439+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:26:30.443+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:26:30.442+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:26:30.443+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:26:30.462+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.037 seconds
[2025-06-19T01:27:00.832+0000] {processor.py:161} INFO - Started process (PID=52087) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:27:00.833+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:27:00.834+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:27:00.834+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:27:00.837+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:27:00.836+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:27:00.838+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:27:00.854+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.031 seconds
[2025-06-19T01:27:31.421+0000] {processor.py:161} INFO - Started process (PID=52123) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:27:31.423+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:27:31.424+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:27:31.423+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:27:31.427+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:27:31.426+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:27:31.428+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:27:31.444+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.032 seconds
[2025-06-19T01:28:02.059+0000] {processor.py:161} INFO - Started process (PID=52159) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:28:02.061+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:28:02.062+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:28:02.061+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:28:02.064+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:28:02.063+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:28:02.065+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:28:02.083+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.033 seconds
[2025-06-19T01:28:32.692+0000] {processor.py:161} INFO - Started process (PID=52195) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:28:32.693+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:28:32.694+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:28:32.693+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:28:32.697+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:28:32.696+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:28:32.697+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:28:32.715+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.033 seconds
[2025-06-19T01:29:03.620+0000] {processor.py:161} INFO - Started process (PID=52231) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:29:03.622+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:29:03.624+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:29:03.624+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:29:03.630+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:29:03.628+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:29:03.632+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:29:03.665+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.056 seconds
[2025-06-19T01:29:33.727+0000] {processor.py:161} INFO - Started process (PID=52266) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:29:33.728+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:29:33.730+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:29:33.729+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:29:33.734+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:29:33.733+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:29:33.734+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:29:33.755+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.039 seconds
[2025-06-19T01:30:03.909+0000] {processor.py:161} INFO - Started process (PID=52302) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:30:03.911+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:30:03.913+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:30:03.913+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:30:03.917+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:30:03.916+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:30:03.918+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:30:03.940+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.040 seconds
[2025-06-19T01:30:34.562+0000] {processor.py:161} INFO - Started process (PID=52339) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:30:34.563+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:30:34.564+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:30:34.564+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:30:34.568+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:30:34.566+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:30:34.568+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:30:34.584+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.031 seconds
[2025-06-19T01:31:05.137+0000] {processor.py:161} INFO - Started process (PID=52375) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:31:05.142+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:31:05.144+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:31:05.143+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:31:05.149+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:31:05.147+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:31:05.149+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:31:05.169+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.042 seconds
[2025-06-19T01:31:35.943+0000] {processor.py:161} INFO - Started process (PID=52411) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:31:35.944+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:31:35.945+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:31:35.945+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:31:35.948+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:31:35.947+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:31:35.949+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:31:35.973+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.040 seconds
[2025-06-19T01:32:06.566+0000] {processor.py:161} INFO - Started process (PID=52447) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:32:06.567+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:32:06.568+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:32:06.568+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:32:06.573+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:32:06.571+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:32:06.578+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:32:06.598+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.044 seconds
[2025-06-19T01:32:37.300+0000] {processor.py:161} INFO - Started process (PID=52483) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:32:37.301+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:32:37.302+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:32:37.302+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:32:37.307+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:32:37.305+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:32:37.308+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:32:37.328+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.039 seconds
[2025-06-19T01:33:07.902+0000] {processor.py:161} INFO - Started process (PID=52519) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:33:07.903+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:33:07.904+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:33:07.904+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:33:07.908+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:33:07.907+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:33:07.909+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:33:07.929+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.036 seconds
[2025-06-19T01:33:38.589+0000] {processor.py:161} INFO - Started process (PID=52555) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:33:38.590+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:33:38.591+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:33:38.591+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:33:38.594+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:33:38.593+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:33:38.595+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:33:38.615+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.036 seconds
[2025-06-19T01:34:09.392+0000] {processor.py:161} INFO - Started process (PID=52591) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:34:09.393+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:34:09.395+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:34:09.394+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:34:09.399+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:34:09.398+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:34:09.400+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:34:09.424+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.044 seconds
[2025-06-19T01:34:40.068+0000] {processor.py:161} INFO - Started process (PID=52627) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:34:40.070+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:34:40.071+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:34:40.071+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:34:40.075+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:34:40.074+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:34:40.076+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:34:40.095+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.037 seconds
[2025-06-19T01:35:10.820+0000] {processor.py:161} INFO - Started process (PID=52664) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:35:10.822+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:35:10.823+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:35:10.823+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:35:10.827+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:35:10.826+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:35:10.828+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:35:10.848+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.038 seconds
[2025-06-19T01:35:41.642+0000] {processor.py:161} INFO - Started process (PID=52700) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:35:41.643+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:35:41.644+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:35:41.644+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:35:41.649+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:35:41.648+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:35:41.651+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:35:41.673+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.050 seconds
[2025-06-19T01:36:12.455+0000] {processor.py:161} INFO - Started process (PID=52736) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:36:12.456+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:36:12.457+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:36:12.457+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:36:12.462+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:36:12.461+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:36:12.464+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:36:12.487+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.045 seconds
[2025-06-19T01:36:43.318+0000] {processor.py:161} INFO - Started process (PID=52772) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:36:43.320+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:36:43.321+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:36:43.321+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:36:43.324+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:36:43.323+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:36:43.325+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:36:43.347+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.039 seconds
[2025-06-19T01:37:13.794+0000] {processor.py:161} INFO - Started process (PID=52808) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:37:13.795+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:37:13.797+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:37:13.796+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:37:13.801+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:37:13.800+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:37:13.802+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:37:13.822+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.039 seconds
[2025-06-19T01:37:44.649+0000] {processor.py:161} INFO - Started process (PID=52844) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:37:44.651+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:37:44.652+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:37:44.652+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:37:44.657+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:37:44.655+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:37:44.658+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:37:44.682+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.045 seconds
[2025-06-19T01:38:15.354+0000] {processor.py:161} INFO - Started process (PID=52880) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:38:15.355+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:38:15.356+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:38:15.356+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:38:15.360+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:38:15.358+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:38:15.361+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:38:15.384+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.040 seconds
[2025-06-19T01:38:46.058+0000] {processor.py:161} INFO - Started process (PID=52917) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:38:46.059+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:38:46.060+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:38:46.060+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:38:46.064+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:38:46.062+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:38:46.064+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:38:46.086+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.039 seconds
[2025-06-19T01:39:16.685+0000] {processor.py:161} INFO - Started process (PID=52953) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:39:16.686+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:39:16.687+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:39:16.687+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:39:16.691+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:39:16.690+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:39:16.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:39:16.712+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.037 seconds
[2025-06-19T01:39:47.211+0000] {processor.py:161} INFO - Started process (PID=52989) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:39:47.211+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:39:47.213+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:39:47.212+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:39:47.216+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:39:47.215+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:39:47.217+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:39:47.237+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.037 seconds
[2025-06-19T01:40:17.949+0000] {processor.py:161} INFO - Started process (PID=53025) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:40:17.951+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:40:17.955+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:40:17.955+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:40:17.958+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:40:17.957+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:40:17.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:40:17.982+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.043 seconds
[2025-06-19T01:40:48.573+0000] {processor.py:161} INFO - Started process (PID=53061) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:40:48.575+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:40:48.577+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:40:48.576+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:40:48.580+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:40:48.579+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:40:48.581+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:40:48.604+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.040 seconds
[2025-06-19T01:41:19.290+0000] {processor.py:161} INFO - Started process (PID=53097) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:41:19.291+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:41:19.293+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:41:19.293+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:41:19.297+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:41:19.296+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:41:19.299+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:41:19.324+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.046 seconds
[2025-06-19T01:41:50.083+0000] {processor.py:161} INFO - Started process (PID=53133) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:41:50.084+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:41:50.086+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:41:50.085+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:41:50.090+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:41:50.089+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:41:50.091+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:41:50.115+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.044 seconds
[2025-06-19T01:42:21.221+0000] {processor.py:161} INFO - Started process (PID=53170) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:42:21.222+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:42:21.224+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:42:21.223+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:42:21.229+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:42:21.227+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:42:21.231+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:42:21.257+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.051 seconds
[2025-06-19T01:42:53.237+0000] {processor.py:161} INFO - Started process (PID=53212) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:42:53.239+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:42:53.240+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:42:53.240+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:42:53.245+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:42:53.244+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:42:53.246+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:42:53.273+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.048 seconds
[2025-06-19T01:43:23.893+0000] {processor.py:161} INFO - Started process (PID=53248) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:43:23.895+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:43:23.896+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:43:23.896+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:43:23.900+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:43:23.899+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:43:23.901+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:43:23.929+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.048 seconds
[2025-06-19T01:43:54.168+0000] {processor.py:161} INFO - Started process (PID=53284) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:43:54.169+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:43:54.170+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:43:54.170+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:43:54.175+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:43:54.173+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:43:54.175+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:43:54.201+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.044 seconds
[2025-06-19T01:44:24.386+0000] {processor.py:161} INFO - Started process (PID=53320) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:44:24.387+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:44:24.389+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:44:24.389+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:44:24.395+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:44:24.393+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:44:24.396+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:44:24.423+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.052 seconds
[2025-06-19T01:44:54.730+0000] {processor.py:161} INFO - Started process (PID=53357) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:44:54.732+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:44:54.734+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:44:54.733+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:44:54.738+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:44:54.737+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:44:54.739+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:44:54.765+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.047 seconds
[2025-06-19T01:45:25.035+0000] {processor.py:161} INFO - Started process (PID=53393) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:45:25.036+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:45:25.038+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:45:25.037+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:45:25.043+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:45:25.041+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:45:25.044+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:45:25.068+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.045 seconds
[2025-06-19T01:45:55.908+0000] {processor.py:161} INFO - Started process (PID=53429) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:45:55.910+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:45:55.912+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:45:55.912+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:45:55.918+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:45:55.916+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:45:55.919+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:45:55.951+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.058 seconds
[2025-06-19T01:46:27.214+0000] {processor.py:161} INFO - Started process (PID=53465) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:46:27.232+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:46:27.235+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:46:27.235+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:46:27.250+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:46:27.247+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:46:27.267+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:46:27.320+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.123 seconds
[2025-06-19T01:46:57.895+0000] {processor.py:161} INFO - Started process (PID=53501) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:46:57.902+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:46:57.904+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:46:57.903+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:46:57.909+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:46:57.907+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:46:57.911+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:46:57.946+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.065 seconds
[2025-06-19T01:47:28.278+0000] {processor.py:161} INFO - Started process (PID=53537) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:47:28.280+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:47:28.282+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:47:28.282+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:47:28.287+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:47:28.286+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:47:28.288+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:47:28.315+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.050 seconds
[2025-06-19T01:47:58.549+0000] {processor.py:161} INFO - Started process (PID=53573) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:47:58.551+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:47:58.553+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:47:58.552+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:47:58.559+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:47:58.557+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:47:58.561+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:47:58.596+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.062 seconds
[2025-06-19T01:48:28.917+0000] {processor.py:161} INFO - Started process (PID=53609) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:48:28.919+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:48:28.921+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:48:28.921+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:48:28.926+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:48:28.925+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:48:28.927+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:48:28.955+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.051 seconds
[2025-06-19T01:48:59.179+0000] {processor.py:161} INFO - Started process (PID=53645) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:48:59.180+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:48:59.182+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:48:59.181+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:48:59.185+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:48:59.184+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:48:59.186+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:48:59.212+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.046 seconds
[2025-06-19T01:49:29.683+0000] {processor.py:161} INFO - Started process (PID=53681) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:49:29.685+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:49:29.687+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:49:29.686+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:49:29.692+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:49:29.691+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:49:29.693+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:49:29.722+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.052 seconds
[2025-06-19T01:49:59.832+0000] {processor.py:161} INFO - Started process (PID=53717) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:49:59.834+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:49:59.835+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:49:59.835+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:49:59.840+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:49:59.839+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:49:59.843+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:49:59.868+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.047 seconds
[2025-06-19T01:50:30.307+0000] {processor.py:161} INFO - Started process (PID=53753) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:50:30.309+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:50:30.310+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:50:30.310+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:50:30.316+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:50:30.315+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:50:30.318+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:50:30.344+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.049 seconds
[2025-06-19T01:51:00.445+0000] {processor.py:161} INFO - Started process (PID=53789) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:51:00.447+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:51:00.449+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:51:00.449+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:51:00.454+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:51:00.453+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:51:00.455+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:51:00.480+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.048 seconds
[2025-06-19T01:51:31.063+0000] {processor.py:161} INFO - Started process (PID=53826) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:51:31.065+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:51:31.066+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:51:31.066+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:51:31.072+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:51:31.070+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:51:31.074+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:51:31.097+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.046 seconds
[2025-06-19T01:52:01.828+0000] {processor.py:161} INFO - Started process (PID=53862) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:52:01.830+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:52:01.831+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:52:01.831+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:52:01.836+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:52:01.834+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:52:01.837+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:52:01.864+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.048 seconds
[2025-06-19T01:52:32.281+0000] {processor.py:161} INFO - Started process (PID=53898) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:52:32.282+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:52:32.283+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:52:32.283+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:52:32.287+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:52:32.286+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:52:32.288+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:52:32.307+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.036 seconds
[2025-06-19T01:53:02.761+0000] {processor.py:161} INFO - Started process (PID=53934) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:53:02.762+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:53:02.764+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:53:02.763+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:53:02.769+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:53:02.767+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:53:02.773+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:53:02.796+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.061 seconds
[2025-06-19T01:53:32.955+0000] {processor.py:161} INFO - Started process (PID=53970) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:53:32.956+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:53:32.957+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:53:32.956+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:53:32.961+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:53:32.959+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:53:32.962+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:53:32.980+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.034 seconds
[2025-06-19T01:54:03.222+0000] {processor.py:161} INFO - Started process (PID=54007) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:54:03.223+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:54:03.225+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:54:03.225+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:54:03.228+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:54:03.227+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:54:03.229+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:54:03.249+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.036 seconds
[2025-06-19T01:54:33.695+0000] {processor.py:161} INFO - Started process (PID=54043) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:54:33.696+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:54:33.697+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:54:33.697+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:54:33.701+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:54:33.699+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:54:33.701+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:54:33.720+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.034 seconds
[2025-06-19T01:55:04.117+0000] {processor.py:161} INFO - Started process (PID=54079) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:55:04.118+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:55:04.120+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:55:04.119+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:55:04.123+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:55:04.122+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:55:04.124+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:55:04.142+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.036 seconds
[2025-06-19T01:55:34.317+0000] {processor.py:161} INFO - Started process (PID=54115) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:55:34.331+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:55:34.334+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:55:34.333+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:55:34.338+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:55:34.337+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:55:34.347+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:55:34.364+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.055 seconds
[2025-06-19T01:56:04.637+0000] {processor.py:161} INFO - Started process (PID=54151) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:56:04.638+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:56:04.639+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:56:04.639+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:56:04.643+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:56:04.642+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:56:04.645+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:56:04.667+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.039 seconds
[2025-06-19T01:56:34.933+0000] {processor.py:161} INFO - Started process (PID=54187) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:56:34.935+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:56:34.936+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:56:34.935+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:56:34.939+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:56:34.938+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:56:34.941+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:56:34.959+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.036 seconds
[2025-06-19T01:57:05.227+0000] {processor.py:161} INFO - Started process (PID=54223) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:57:05.228+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:57:05.229+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:57:05.229+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:57:05.233+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:57:05.232+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:57:05.233+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:57:05.251+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.034 seconds
[2025-06-19T01:57:35.806+0000] {processor.py:161} INFO - Started process (PID=54259) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:57:35.808+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:57:35.810+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:57:35.809+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:57:35.814+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:57:35.813+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:57:35.815+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:57:35.837+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.042 seconds
[2025-06-19T01:58:06.154+0000] {processor.py:161} INFO - Started process (PID=54295) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:58:06.155+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:58:06.156+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:58:06.156+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:58:06.160+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:58:06.159+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:58:06.161+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:58:06.182+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.038 seconds
[2025-06-19T01:58:36.515+0000] {processor.py:161} INFO - Started process (PID=54331) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:58:36.516+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:58:36.517+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:58:36.517+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:58:36.521+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:58:36.520+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:58:36.522+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:58:36.540+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.035 seconds
[2025-06-19T01:59:06.830+0000] {processor.py:161} INFO - Started process (PID=54367) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:59:06.830+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:59:06.831+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:59:06.831+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:59:06.835+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:59:06.834+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:59:06.836+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:59:06.856+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.035 seconds
[2025-06-19T01:59:37.303+0000] {processor.py:161} INFO - Started process (PID=54403) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:59:37.304+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T01:59:37.306+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:59:37.305+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:59:37.309+0000] {logging_mixin.py:188} INFO - [2025-06-19T01:59:37.308+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T01:59:37.311+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T01:59:37.331+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.037 seconds
[2025-06-19T02:00:07.631+0000] {processor.py:161} INFO - Started process (PID=54439) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T02:00:07.632+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T02:00:07.633+0000] {logging_mixin.py:188} INFO - [2025-06-19T02:00:07.633+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T02:00:07.637+0000] {logging_mixin.py:188} INFO - [2025-06-19T02:00:07.636+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T02:00:07.638+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T02:00:07.653+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.030 seconds
[2025-06-19T02:00:37.897+0000] {processor.py:161} INFO - Started process (PID=54475) to work on /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T02:00:37.898+0000] {processor.py:830} INFO - Processing file /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py for tasks to queue
[2025-06-19T02:00:37.899+0000] {logging_mixin.py:188} INFO - [2025-06-19T02:00:37.899+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T02:00:37.903+0000] {logging_mixin.py:188} INFO - [2025-06-19T02:00:37.902+0000] {dagbag.py:355} ERROR - Failed to import: /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py", line 22, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/__init__.py", line 37, in <module>
    raise RuntimeError(
RuntimeError: The package `apache-airflow-providers-standard:1.3.0` needs Apache Airflow 2.10.0+
[2025-06-19T02:00:37.905+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py
[2025-06-19T02:00:37.922+0000] {processor.py:183} INFO - Processing /home/airflow/.local/lib/python3.12/site-packages/airflow/example_dags/standard/example_external_task_child_deferrable.py took 0.034 seconds
